### ISSUE TITLE
Unhandled exception 

### ISSUE THREAD

**Issue by krash50** [securestate/king-phisher#251]
## Issue Description
I get an unhandled exception when logging into the server (Windows).  Error type: builtins.ValueError

## Reproduction Steps
1. Start King Phisher
2. Login to the server

## Environment Details

Host OS: Windows: 10 10.0.15063  Multiprocessor Free
King Phisher Version: 1.9.0

## Error Details / Stack Trace

```
Error Type: builtins.ValueError
Error Details: ValueError('too many values to unpack (expected 2)',)
Error UID: 5eaeb50f-a86c-4ccd-aca8-598e6e436029
RPC Error: N/A (Not a remote RPC error)
King Phisher Version: 1.9.0
Platform Version: Windows: 10 10.0.15063  Multiprocessor Free (Frozen=True)
Python Version: 3.4.4
Gtk Version: 3.14.15
Timezone: America/New_York

Thread Information:
 => MainThread (alive=True daemon=False)

Stack Trace:
Traceback (most recent call last):
  File "C:\king-phisher\king_phisher\client\application.py", line 573, in server_connect
  File "C:\Python34\lib\site-packages\advancedhttpserver.py", line 550, in __call__
  File "C:\Python34\lib\site-packages\advancedhttpserver.py", line 596, in call
  File "C:\Python34\lib\http\client.py", line 1137, in request
  File "C:\Python34\lib\http\client.py", line 1182, in _send_request
  File "C:\Python34\lib\http\client.py", line 1133, in endheaders
  File "C:\Python34\lib\http\client.py", line 963, in _send_output
  File "C:\Python34\lib\http\client.py", line 898, in send
  File "C:\Python34\lib\http\client.py", line 871, in connect
  File "C:\Python34\lib\socket.py", line 516, in create_connection
  File "C:\Python34\lib\socket.py", line 507, in create_connection
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\king-phisher\king_phisher\client\windows\main.py", line 313, in signal_login_dialog_response
  File "C:\king-phisher\king_phisher\client\application.py", line 584, in server_connect
  File "C:\king-phisher\king_phisher\client\gui_utilities.py", line 516, in show_dialog_exc_socket_error
ValueError: too many values to unpack (expected 2)

```

**Comment by zeroSteiner** [comment]
Thanks for reporting this @krash50. I'm reasonably certain that the issue here is that `show_dialog_exc_socket_error` is running into trouble formatting the exception for display to the user (you).

We'll get this resolved but in the mean time you'll want to look at the server connection settings as even after the fix has been implemented it will still have trouble connecting. You'll want to make sure that the King Phisher server is up and running on the target server. Check out the FAQ question [The King Phisher Client Fails To Connect](https://github.com/securestate/king-phisher/wiki/Frequently-Asked-Questions#king-phisher-client-fails-to-connect) for some more information.

**Comment by krash50** [comment]
Thank you.  The software was too much of a pain to configure to run in a Windows environment so I am going back to what I already had setup in Linux.

**Comment by zeroSteiner** [comment]
This issue has now been fixed in the dev branch. To obtain the changes, you can either wait for them to be merged into the master branch (which will likely take a few weeks) or you can obtain them immediately by opting into [beta testing](https://github.com/securestate/king-phisher/wiki/Updating-King-Phisher#beta-testing).

Fixed in commit 671f946b28df98800d7f13cccf77e91c5e6cbdee.
